starknet_py/net/client_models.py

```diff
@@ -178,7 +178,7 @@ class DeclareTransactionResponse(SentTransactionResponse):
     Dataclass representing a result of declaring a contract on starknet
     """
 
-    class_hash: int = None
```

Comment:
I wonder if we should be having default values here? Perhaps this should be resolved on schema level instead?

Reply:
SentTransactionResponse has default value so DeclareTransactionResponse must have it too

Reply:
Could't it be changed? I'm not 100% sure what's the point of having keeping defaults in dataclasses instead of schemas.

Reply:
![image](https://user-images.githubusercontent.com/10278469/191934262-35637318-abdc-490c-b699-5b64307484b9.png)

Reply:
You can change this in a seperate PR if you want. This PR has a different purpose.

Reply:
Okay, this can be resolved in a separate PR